diff --git a/Monicelli.ypp b/Monicelli.ypp
index 028506d..9817f1e 100644
--- a/Monicelli.ypp
+++ b/Monicelli.ypp
@@ -23,7 +23,7 @@
 }
 
 %skeleton "lalr1.cc"
-%require "3.0"
+%require "2.5"
 %language "c++"
 
 %defines
@@ -31,9 +31,11 @@
 %locations
 %token-table
 
-%define parse.error verbose
-%define api.namespace {monicelli}
-%define parser_class_name {Parser}
+%{
+#define YYERROR_VERBOSE
+%}
+%define namespace monicelli
+%define parser_class_name Parser
 
 %lex-param {Scanner &scanner}
 %parse-param {Scanner &scanner}
